### PR TITLE
Add min_count parameter for Frame.sum.

### PR DESCRIPTION
**Pull request by ueshin**
Adds `min_count` parameter for `Frame.sum`.

### CODE REVIEW

File: databricks/koalas/frame.py [hunk 1]
```diff
@@ -632,21 +632,27 @@ def _reduce_for_stat_function(self, sfun, name, axis=None, numeric_only=True):
             new_column_labels = []
             num_args = len(signature(sfun).parameters)
             for label in self._internal.column_labels:
-                col_sdf = self._internal.spark_column_for(label)
-                col_type = self._internal.spark_type_for(label)
+                spark_column = self._internal.spark_column_for(label)
+                spark_type = self._internal.spark_type_for(label)
```
Comment (itholic) [inline databricks/koalas/frame.py:636]: +1

**Review by itholic — APPROVED** [review]
Cool. LGTM

**Comment by xinrong-databricks** [comment]
Are we going to add test cases in a separate pr?

**Review by xinrong-databricks — APPROVED** [review]

**Comment by ueshin** [comment]
I'll fix some corner case and add tests soon.

**Comment by xinrong-databricks** [comment]
Great! LGTM!

**Comment by ueshin** [comment]
Thanks! merging.
